repos/spack_repo/builtin/packages/openblas/package.py

```diff
@@ -242,6 +242,8 @@ class Openblas(CMakePackage, MakefilePackage):
         when="@0.3.27 %oneapi",
     )
 
+    conflicts("%fortran=clang")
```

Comment:
This might be too restrictive. `-mtune` seems to exists in `flang-new` in LLVM 19+.

Reply:
Seems to first exist in 19.1: https://github.com/llvm/llvm-project/pull/98517

Reply:
I actually wonder which version of OpenBLAS added `-mtune`, we could be even less restrictive.